hindi/Akshay_Kumar/akshay_kumar.md

```diff
@@ -0,0 +1,6 @@
+Here are some facts given about Akshay Kumar, to know more click [here](https://en.wikipedia.org/wiki/Akshay_Kumar)
+Rajiv Hari Om Bhatia (born 9 September 1967), better known by his stage name Akshay Kumar, is an Indian-born Canadian[1] actor, producer and martial artist who has appeared in over a hundred Hindi films.[2] He has been nominated for Filmfare Awards several times, winning it two times. When he began his acting career in the 1990s, he primarily starred in action films and was known for his appearances in films of the "Khiladi series" as well as other action films such as Waqt Hamara Hai (1993), Mohra (1994), Elaan (1994), Suhaag (1994), Sapoot (1996), Jaanwar (1999).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.